src/dndbeyond/content-scripts/character.js

```diff
@@ -2312,8 +2312,8 @@ function deactivateQuickRolls() {
     const spells = $(".ct-spells-spell .ct-spells-spell__action,.ddbc-spells-spell .ddbc-spells-spell__action");
     const spells_to_hit = $(".ct-spells-spell .ct-spells-spell__tohit .integrated-dice__container, .ddbc-spells-spell .ddbc-spells-spell__tohit .integrated-dice__container");
     const spells_damage = $(".ct-spells-spell .ct-spells-spell__damage .integrated-dice__container, .ddc-spells-spell .ddc-spells-spell__damage .integrated-dice__container");
-    const initiative = $(".ct-combat__summary-group--initiative .integrated-dice__container, .ct-combat__summary-group--initiative span[class*='styles_numberDisplay'], .ct-combat-tablet__extra--initiative .integrated-dice__container, .ct-combat-tablet__extra--initiative span[class*='styles_numberDisplay'], .ct-combat-mobile__extras > section[class*='styles_boxMobile'] span[class*='styles_numberDisplay']");
-
+    const initiative = $(".ct-combat__summary-group--initiative .integrated-dice__container, .ct-combat-tablet__extra--initiative span[class*='styles_numberDisplay'], .ct-combat-mobile__extras > section[class*='styles_boxMobile'] span[class*='styles_numberDisplay']");
```

Comment:
ah, good catch, thanks!
I think the integrated dice container also contained that `span[class*='styles_numberDisplay']`, that's why it showed up twice. One possibility would be to do `.eq(0)` at the end of the request so it just picks the first one, or remove the duplicates like you did. However, you removed the span, and kept integrated-dice__container, so you fixed the double rolls when digital dice are on, and it breaks when digital dice are disabled.